proteus/mprans/VOF.py

```diff
@@ -416,18 +418,18 @@ def attachModels(self,modelList):
             #
         #
     def initializeElementQuadrature(self,t,cq):
-        if self.flowModelIndex is None:
+        if self.flowModelIndex == None:
```

Comment:
switch to "is None", see e.g. http://jaredgrubb.blogspot.com/2009/04/python-is-none-vs-none.html

Reply:
I can take care of the file management/version control. The .h5 file will need to be purged from the git history to keep the repo size small.